domain/schema/controller.go

```diff
@@ -542,7 +542,7 @@ CREATE TABLE user (
     name            TEXT NOT NULL,
     display_name    TEXT,
     removed         BOOLEAN NOT NULL,
```

Comment:
I think we should possible add to `removed` with `DEFAULT FALSE`

Reply:
+